compiler/optimizer/DataFlowAnalysis.enum

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2000, 2018 IBM Corp. and others
+ * Copyright (c) 2018, 2018 IBM Corp. and others
```

Comment:
again I am wondering why the initial copyright date was changed?

Reply:
git didnt really track the changes properly. DataFlowAnalysis,enum was renamed to OMRDataFlowAnalysis.enum, and  a new file was created with the old name. I was hoping that by keeping the rename in one commit, and the creation of the new file in a separate commit, git would properly track the file move, but that seems to have failed